code/modules/mob/living/carbon/xenomorph/Castes/Boiler.dm

```diff
@@ -280,3 +284,8 @@
 	bomb_cooldown = FALSE
 	to_chat(src, "<span class='notice'>You feel your toxin glands swell. You are able to bombard an area again.</span>")
 	update_action_button_icons()
+
+/mob/living/carbon/Xenomorph/Boiler/Bumped(atom/movable/AM as mob|obj)
+	if(emitting_gas) //We don't get bumped around
```

Comment:
Do `anchored = TRUE` when emitting, and `anchored = FALSE` when not, instead, please.

Reply:
I really don't like using anchored as it can get wonky but sure